js/core/dom_component.d.ts

```diff
@@ -8,14 +8,26 @@ import {
     DxElement,
 } from './element';
 
+import {
+    ChangedOptionInfo,
+    EventInfo,
+    InitializedEventInfo,
+} from '../events/index';
+
 import { TemplateManager } from './template_manager';
 import { FunctionTemplate } from './templates/function_template';
 import { DefaultOptionsRule } from './options';
 
+export type OptionChangedEventInfo<TComponent> = EventInfo<TComponent> & ChangedOptionInfo;
```

Comment:
As far as I can see, this type is not used anywhere else
```suggestion
type OptionChangedEventInfo<TComponent> = EventInfo<TComponent> & ChangedOptionInfo;
```